Formula/nlohmann_json.rb

```diff
@@ -0,0 +1,65 @@
+class NlohmannJson < Formula
+  desc "JSON for Modern C++"
+  homepage "https://github.com/nlohmann/json"
+  url "https://github.com/nlohmann/json/archive/v1.1.0.tar.gz"
+  sha256 "ee3825841e6d6915428caf2cea53927c3e4d56315a23ee7d1a64bfe1c19a656f"
+  head "https://github.com/nlohmann/json.git"
+
+  def install
+    include.install "src/json.hpp"
+    ohai "to use the library, please set your include path accordingly:"
```

Comment:
Should the message be removed altogether our just be displayed with a command different than `ohai`?